src/test/java/com/sap/oss/phosphor/fosstars/model/score/oss/ArtifactReleaseHistoryScoreTest.java

```diff
@@ -130,49 +146,44 @@ public void testTwoMonthOldVersion() {
     Assert.assertEquals(7.0, value.get(), DELTA);
   }
 
-  /**
-   * Temporarily ignoring this test method.
-   * 
-   * @see <a href="https://github.com/SAP/fosstars-rating-core/issues/511">Issue 511</a>
-   */
   @Test
-  @Ignore
   public void testTwoMonthOldVersionStable() {
+    LocalDateTime now = LocalDateTime.now();
     ArtifactVersion version100 =
-        new ArtifactVersion("1.0.0", LocalDate.now().minusMonths(7));
+        new ArtifactVersion("1.0.0", now.minusDays(7 * 31));
```

Comment:
The issue only occurs in cases in which the test releases had one month difference.
However, to be on the very safe side it may be better to replace all by `minusDays`.